src/client.rs

```diff
@@ -60,19 +60,70 @@ fn default_hash_function(key: &str) -> u64 {
     return hasher.finish();
 }
 
-impl Client {
-    #[deprecated(since = "0.10.0", note = "please use `connect` instead")]
-    pub fn new<C: Connectable>(target: C) -> Result<Self, MemcacheError> {
-        return Self::connect(target);
+pub struct ClientBuilder<C> {
```

Comment:
This struct will be used more often, I think we should re-export it to crate's top level.

Reply:
Yes, will do when I find some time.